src/dbt/kipptaf/models/extracts/powerschool/rpt_powerschool__autocomm_students.sql

```diff
@@ -1,3 +1,24 @@
+with
+    grad_path as (
+        select _dbt_source_relation, student_number, discipline, final_grad_path,
+        from {{ ref("int_students__graduation_path_codes") }}
+    ),
+
+    grad_path_pivot as (
+        select
+            _dbt_source_relation,
+            student_number,
+            s_nj_stu_x__graduation_pathway_math,
+            s_nj_stu_x__graduation_pathway_ela,
+        from
+            grad_path pivot (
+                max(final_grad_path) for discipline in (
+                    'Math' as `s_nj_stu_x__graduation_pathway_math`,
+                    'ELA' as `s_nj_stu_x__graduation_pathway_ela`
+                )
+            )
+    )
```

Comment:
@GabyRangelB here's an example of using the `pivot` operator

Reply:
I tried doing that, but I was getting double rows for students when the pathway letters were not both the same.

Reply:
So, in other words, I was getting one line for Math where math was filled but ELA was blank, and then one line for ELA where ELA was filed and math was blank.

Reply:
yeah I had to split it up into 2 CTEs to make it pivot correctly. not entirely sure why that happens sometimes